queue/test_bootstrapper.go

```diff
@@ -2,11 +2,33 @@
 
 package queue
 
+import (
+	"github.com/centrifuge/go-centrifuge/bootstrap"
+	"sync"
+	"context"
+)
+
 func (b *Bootstrapper) TestBootstrap(context map[string]interface{}) error {
 	return b.Bootstrap(context)
 }
 
 func (b *Bootstrapper) TestTearDown() error {
-	StopQueue()
+	srv := b.context[bootstrap.BootstrappedQueueServer].(*Server)
+	return srv.Stop()
+}
+
+type Starter struct {}
```

Comment:
Not sure I understand the Starter here.

Reply:
server has  two stages of initiation, bootstrap and start. This bootstrapper is to start the queue for tests.